plugins/chat/lib/endpoint.rb

```diff
@@ -2,11 +2,11 @@
 #
 # = Chat::Endpoint
```

Comment:
Not sure `Endpoint` makes any sense any more. Maybe something like `ServiceRunner` or `ServiceWrapper`...maybe `ServiceResponder`? Tricky to name :thinking:

Reply:
I think `actions` is still generic enough

Reply:
Yes, not easy but I think `ServiceRunner` might be a good one, at least it’s quite explicit :grin: